src/main/kotlin/calculator/Operation.kt

```diff
@@ -1,16 +1,13 @@
 package calculator
 
-import java.lang.IllegalArgumentException
-import java.util.function.BiFunction
-
-enum class Operation(val char: String, val operate: BiFunction<Int, Int, Int>) {
-    PLUS("+", BiFunction { first: Int, second: Int -> first + second }),
-    MINUS("-", BiFunction { first: Int, second: Int -> first - second }),
-    MULTIPLE("*", BiFunction { first: Int, second: Int -> first * second }),
-    DIVIDE("/", BiFunction { first: Int, second: Int -> first / second });
+enum class Operation(val char: String, val operate: (first: Int, second: Int) -> Int) {
+    PLUS("+", { first: Int, second: Int -> first + second }),
+    MINUS("-", { first: Int, second: Int -> first - second }),
+    MULTIPLE("*", { first: Int, second: Int -> first * second }),
+    DIVIDE("/", { first: Int, second: Int -> first / second });
 
     fun calc(first: Int, second: Int): Int {
-        return operate.apply(first, second)
+        return operate.invoke(first, second)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return operate(first, second)
```

Reply:
오. 훨씬 깔끔하고 함수 느낌이 확 나는군요 감사합니다!